CHANGES.rst

```diff
@@ -1,3 +1,50 @@
+0.26.0 (2021-03-30)
```

Comment:
It is not really a big deal, but you could make use of the "`" to make certain words render as a code-like keyword, e.g.. ``astroquery.Mast`` instead of astroquery.mast

Reply:
Good to know, thanks!